pkg/process/checks/chunking.go

```diff
@@ -6,18 +6,25 @@
 package checks
 
 import (
+	"os"
+
 	model "github.com/DataDog/agent-payload/v5/process"
 
 	"github.com/DataDog/datadog-agent/pkg/process/util"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
 )
 
 // chunkProcessesBySizeAndWeight chunks `model.Process` payloads by max allowed size and max allowed weight of a chunk
 func chunkProcessesBySizeAndWeight(procs []*model.Process, ctr *model.Container, maxChunkSize, maxChunkWeight int, chunker *util.ChunkAllocator[model.CollectorProc, *model.Process]) {
 	if ctr != nil && len(procs) == 0 {
-		// can happen in two scenarios, and we still need to report the container
+		// can happen in three scenarios, and we still need to report the container
 		// a) if a process is skipped (e.g. disallowlisted)
-		// b) if process <=> container mapping cannot be established (e.g. Docker on Windows)
-		appendContainerWithoutProcesses(ctr, chunker.GetChunks())
+		// b) if process <=> container mapping cannot be established (e.g. Docker on Windows).
+		// c) pidMode not set to "task" on ECS Fargate
```

Comment:
```suggestion
		// c) if no processes were collected from the container (e.g. pidMode not set to "task" on ECS Fargate)
```

Optional, I'd recommend generalizing the scenario in case it comes up in other situations as well

Reply:
Done